mm2src/mm2_main/src/lp_swap/maker_swap.rs

```diff
@@ -1926,19 +1926,33 @@ impl MakerSavedSwap {
         }
     }
 
+    // TODO: Adjust for private coins when/if they are braodcasted
+    // TODO: Adjust for HD wallet when completed
     pub fn swap_pubkeys(&self) -> Result<SwapPubkeys, String> {
-        match self.events.first() {
+        let mut pubkeys = SwapPubkeys {
+            maker: None,
+            taker: None,
+        };
+
+        match &self.events.first() {
             Some(event) => match &event.event {
-                // TODO: Adjust for private coins when/if they are braodcasted
```

Comment:
Please keep this todo since `maker_coin_htlc_pubkey`/`taker_coin_htlc_pubkey` are different if privacy coins are used. Keep also the HD wallet todo.